src/Kongverge/Workflow/Workflow.cs

```diff
@@ -28,6 +28,27 @@ public async Task<int> Execute()
                 return ExitWithCode.Return(ExitCode.HostUnreachable, e.Message);
             }
 
+            var versionSubStr = KongConfiguration.Version.Substring(0, 4);
```

Comment:
Seems fragile...

Reply:
But looking at the format, it seems like it's easier than assuming it's semver and trying to drag in a whole parsing library 🤣